pom.xml

```diff
@@ -2065,6 +2065,11 @@
                 <artifactId>qdox</artifactId>
                 <version>${qdox.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.typesafe.play</groupId>
+                <artifactId>play-json_${scala.base}</artifactId>
+                <version>2.7.4</version>
```

Comment:
you probably need 2.8.1 for value type automatic (de)serialization

Reply:
Okay, I will keep that version for mailbox-event module as I mentioned before, 2.8.1 breaks some scala code inside that module.

Reply:
I think I remember there was an attempt to upgrade it in that `mailbox-event` module to 2.8.1 before. Was it reversed because of something to do with Eclipse? @remk

Reply:
It happens with Intelijj too, IDK whether they are the same problem but for the moment I rather do not touch to that mailbox-event module